editor/src/core/shared/redux-devtools.ts

```diff
@@ -107,25 +112,37 @@ export function reduxDevtoolsSendActions(
 }
 
 export function reduxDevtoolsSendInitialState(newStore: EditorStore): void {
-  if (maybeDevTools != null && sendActionUpdates) {
+  if (maybeDevTools != null) {
```

Comment:
Oooh is this expensive? If it is, we should have this behind the feature flag and have some way of calling it if the feature flag is later enabled

Reply:
no, this is actually fine to happen once. it's a couple of ms but only happens once!